main/src/mill/main/MainModule.scala

```diff
@@ -173,4 +179,34 @@ trait MainModule extends mill.Module{
       }
     }
   }
+
+  /**
+    * Deletes the given targets from the out directory. Providing no targets will clean everything.
+    */
+  def clean(targets: String*) = mill.T.command {
+    val result = Try {
+      if (ammonite.ops.ls(ammonite.ops.pwd).contains(ammonite.ops.pwd / BuildFile)) {
```

Comment:
This can just be `exists(pwd / "build.sc")`

Reply:
Will do.